kvmagent/kvmagent/plugins/ovs_provision_plugin.py

```
@@ -1285,79 +1285,106 @@ def deprovision(self, req):
-                     'ovn-bfd-min-tx', 'ovn-bfd-min-rx', 'ovn-bfd-mult']
-        failed_keys = []
-        for key in ovn_keys:
-            err, val = vsctl.getOvsExternalIdsConfig(key)
-            if not err and val is not None:
-                # For map-type columns, only key is needed to remove the entry
-                r, o, e = bash.bash_roe('ovs-vsctl remove Open_vSwitch . external_ids %s'
-                                        % key)
+        try:
+            # helper: check if a systemd service is active
+            def _is_service_active(svc):
+                r, o, e = bash.bash_roe('systemctl is-active %s' % svc)
```

Ruff (0.15.6)

[warning] Unpacked variable `r` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)